### PR TITLE
Update ZHA service descriptions

### PR DESCRIPTION
**Pull request by Adminiuga**
## Proposed change

https://github.com/home-assistant/core/pull/40652 introduces support to join devices using install codes. Update service descriptions

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/40652
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by dmulcahey — APPROVED** [review]

**Comment by Adminiuga** [comment]
Tested and confirmed QR codes from Consciot and Aqara to work on EmberZnet 6.6.5 and later.

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -144,6 +144,32 @@ enable_quirks:
 
 To add new devices to the network, call the `permit` service on the `zha` domain. Do this by clicking the Service icon in Developer tools and typing `zha.permit` in the **Service** dropdown box. Next, follow the device instructions for adding, scanning or factory reset.
 
+
+## Services
```
Comment (frenck) [inline source/_integrations/zha.markdown:148]: There is an empty line missing after the heading

File: source/_integrations/zha.markdown [hunk 2]
```diff
@@ -144,6 +144,32 @@ enable_quirks:
...
+### Service `permit`
```
Comment (frenck) [inline source/_integrations/zha.markdown:149]: This should be `zha.permit` ?

Also, add a new line after starting a header

File: source/_integrations/zha.markdown [hunk 3]
```diff
@@ -144,6 +144,32 @@ enable_quirks:
...
+This service opens network for joining new devices
```
Comment (frenck) [inline source/_integrations/zha.markdown:150]: Missing full stop at the end of the sentence.

File: source/_integrations/zha.markdown [hunk 4]
```diff
@@ -144,6 +144,32 @@ enable_quirks:
...
+
+|  Data | Optional | Description |
+| ---- | ---- | ----------- |
+| `duration` |  yes | For how long to allow new devices to join, default 60s
+| `ieee` | yes | allow new devices to join via an existing device
+| `src_ieee` | | The IEEE address of the joining ZB3 device. Use with `install_code`
+| `install_code` | | Install Code of the joining device. Use with `src_ieee`
+| `qr_code` | | QR code containing IEEE and Install Code of the joining ZB3 device
+
+* currently `qr_cde` supports QR Install Codes from:
```
Comment (frenck) [inline source/_integrations/zha.markdown:160]: This asterisk `*` creates a list, I feel like this is meant to be as a "note" indicator?

Sentences should start with a capital letter.

File: source/_integrations/zha.markdown [hunk 5]
```diff
@@ -144,6 +144,32 @@ enable_quirks:
...
+- Aqara
+- Consciot
+- Embrighten 
+
+### Service `remove`
```
Comment (frenck) [inline source/_integrations/zha.markdown:165]: I guess this should be `zha.remove`.

Also, a new line is missing after the header

File: source/_integrations/zha.markdown [hunk 6]
```diff
@@ -144,6 +144,32 @@ enable_quirks:
...
+This service remove an existing device from the network
```
Comment (frenck) [inline source/_integrations/zha.markdown:166]: Missing full stop at the end of the line.

File: source/_integrations/zha.markdown [hunk 7]
```diff
@@ -144,6 +144,32 @@ enable_quirks:
...
+
+|  Data | Optional | Description |
+| ---- | ---- | ----------- |
+| `ieee` | no | What device to remove 
+
```
Comment (frenck) [inline source/_integrations/zha.markdown:171]: An additional unneeded new line here, please remove 1

**Review by frenck — COMMENTED** [review]
@Adminiuga 

Some comments, could you please address those in a new PR? 

Thanks.

**Comment by Adminiuga** [comment]
Doh, yeah I will address those.i meant to join the parent PR :facepalm: had too many tabs open. Sorry about that

File: source/_integrations/zha.markdown (hunk 1 above)
Comment (Adminiuga) [reply source/_integrations/zha.markdown:148]: addressed in https://github.com/home-assistant/home-assistant.io/pull/14655/commits/7ef582ca37815a39a60f49360e963a488f4c3314 lets continue in https://github.com/home-assistant/home-assistant.io/pull/14655

File: source/_integrations/zha.markdown (hunk 2 above)
Comment (Adminiuga) [reply source/_integrations/zha.markdown:149]: addressed in https://github.com/home-assistant/home-assistant.io/pull/14655/commits/7ef582ca37815a39a60f49360e963a488f4c3314 lets continue in https://github.com/home-assistant/home-assistant.io/pull/14655

File: source/_integrations/zha.markdown (hunk 3 above)
Comment (Adminiuga) [reply source/_integrations/zha.markdown:150]: addressed in https://github.com/home-assistant/home-assistant.io/pull/14655/commits/7ef582ca37815a39a60f49360e963a488f4c3314 lets continue in https://github.com/home-assistant/home-assistant.io/pull/14655

**Pull request merged**